src/posthog-core.ts

```diff
@@ -1036,13 +1038,14 @@ export class PostHog {
             properties
         )
 
-        const property_blacklist = this.config.property_blacklist
-        if (_isArray(property_blacklist)) {
-            _each(property_blacklist, function (blacklisted_prop) {
+        // since property_blacklist is deprecated in favor of property_denylist, we merge both of them here
+        const property_denylist = [...this.config.property_blacklist, ...this.config.property_denylist]
+        if (_isArray(property_denylist)) {
```

Comment:
Maybe this check has to be made for both arrays now and only merge inside the `if`? not sure what was the issue before.